### PR TITLE
Update basic-for-improved.html

### PR DESCRIPTION
**Pull request by mgo7**
I think there should be  if(i === cats.length -1 ) - otherwise it will never run this code?

### CODE REVIEW

**Comment by chrisdavidmills** [comment]
You are right, this doesn't work! It is funny, this code is correct in the article, but wrong in the GitHub repo.

Thank you for pointing it out, and providing a fix.

I did have a nitpick about it — the code is correct, but your spacing isn't quite right. Can you change the spacing to 

```
if(i === cats.length - 1)
```

then I'll merge it.

Thanks again!

File: javascript/building-blocks/loops/basic-for-improved.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
     const para = document.querySelector('p');
 
     for(let i = 0; i < cats.length; i++) {
-      if(i === cats.length ) {
+      if(i === cats.length - 1 ) {
```
Comment (escape0707) [inline javascript/building-blocks/loops/basic-for-improved.html]:
```suggestion
      if(i === cats.length - 1) {
```

Comment (chrisdavidmills) [reply javascript/building-blocks/loops/basic-for-improved.html]: Ummm, yup? ;-)

**Comment by chrisdavidmills** [comment]
Perfect, thanks for that. Merging.

**Pull request merged**